cirq-core/cirq/ops/controlled_gate_test.py

```diff
@@ -555,3 +555,13 @@ def num_qubits(self) -> int:
             (0.25, cirq.unitary(cirq.CZ)),
         ],
     )
+
```

Comment:
are we testing the new forms here?  I would add a test for these new cases.

Reply:
I assume you are talking about mixed gates, no?
can you suggest a test(s)?

Reply:
I think he means that there are no tests that pass the new class into the ControlledGate constructor and verify the functionality works as desired. There should be tests that do this, preferably with conditions that were not previously expressible.